app/src/main/java/com/onelist/service/ItemService.kt

```diff
@@ -8,14 +8,8 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
-interface IItemService {
-    //TODO: Create the Interface for the Item Service
-
-    //TODO: Create the method to get all the items
-}
-
-class ItemService : IItemService {
-    suspend fun fetchItems(): List<Item>? {
```

Comment:
I'd keep the interface, for Dependency Inversion and Liskov Substitution principles.